packages/landing-gatsby/src/templates/processTamplate.js

```diff
@@ -0,0 +1,342 @@
+                          <MDReactComponent text={policy.description} />
+                        </ContentScroll>
+                      ))}
+                    </ContentWrapper>
+                    <ContentWrapper active={active === 2}>
+                      <p>
+                        Mandatory roles that are affected or created for this
+                        process to work properly.{' '}
+                      </p>
+                      {roles.map((role) => (
+                        <ContentScrollSmall>
+                          <Heading content={role.title} />
+                          <Text content={role.purpose} />
+                          <Text content={role.domains} />
+                          <Text content={role.accountabilities} />
+                        </ContentScrollSmall>
+                      ))}
+                    </ContentWrapper>
+                    <ContentWrapper active={active === 3}>
+                      <p>Tools that are used in this process.</p>
+                      <FlexWrapper>
+                        {tools.map((tool, index) => (
+                          <ExternalLink href={tool.link}>
+                            <ContentHoverTools key={index}>
+                              <ToolImg src={tool.link} />
+                              <ToolContent>
+                                <Heading content={tool.name} />
+                                <Text content={tool.description} />
+                              </ToolContent>
+                            </ContentHoverTools>
+                          </ExternalLink>
+                        ))}
+                      </FlexWrapper>
+                    </ContentWrapper>
+                    <ContentWrapper active={active === 4}>
+                      <p>
+                        Values drive the organisation and the following are
+                        needed to fully adapt this process.
+                      </p>
+                      {values.map((value) => (
+                        <ContentScroll>
+                          <Heading content={value.name} />
+                          <MDReactComponent text={value.description} />
+                        </ContentScroll>
+                      ))}
+                    </ContentWrapper>
+                  </Fragment>
+                </>
+              </dvi>
+            </SectionWrapper>
+            <SectionWrapper>
+              <YellowHighliter>
+                <Heading content="Integration Steps" />
+              </YellowHighliter>
+              <ContentWrapperIntegration>
+                <StepNavigation>
+                  {pageContent.instructions.map((instruction, index) => (
+                    <StepNavigationElement
+                      active={active === 0 + index}
+                      id={index}
+                      onClick={handleClick}
+                    >
+                      <a href={handleClick}>
```

Comment:
Warning: Invalid value for prop `href` on <a> tag. Either remove it from the element, or pass a string or number value to keep it in the DOM.